sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationUpdateMode.kt

```diff
@@ -0,0 +1,33 @@
+package com.mapbox.maps.viewannotation
+
+/**
+ * Describes modes that could be applied to update view annotation positions and
+ * synchronize them with current map camera.
+ *
+ * Note: behaviour could still be undetermined as we could not control Android render thread directly.
+ */
+enum class ViewAnnotationUpdateMode {
+  /**
+   * View annotations are updated on the next frame comparing to the MapView camera.
+   *
+   * Using this mode introduces fixed 1-frame delay for view annotation position and if panning the map
+   * quickly view annotation will be visually behind the map.
+   */
+  FIXED_DELAY,
+
+  /**
+   * View annotations are updated independent from MapView.
+   * View annotation position could be updated on the same frame as the map camera as well as on next one.
+   *
+   * Using this mode is not recommended as it will most likely bring in view annotation jittering.
+   */
+  MAP_INDEPENDENT,
```

Comment:
@mapbox/maps-android - to discuss if we need it at all. Looking at how it actually behaves I'm leaning to actually cut this enum out but would like to hear your opinion as well

Reply:
Does this mode provide advantages compared with the other two modes? From the video this mode seems to be both laggy and jittering at the same time 🙈

Reply:
I'm 👍 with removing if there's no specific advantages.

Reply:
Did remove it indeed, I can't see any future for this mode with current core implementation. Thanks for your opinion @pengdev!